app/services/locale_service/locale.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class LocaleService::Locale # rubocop:disable Style/ClassAndModuleChildren
```

Comment:
```suggestion
module LocaleService
  class Locale
```

Does this accomplish the same thing and not need the magic rubocop comment?

Reply:
no../. i mean yes, kind of, but it was causing other namespace issue because I want a classnot a module

short answer, I tried to use `module` but couldnt get it working

Reply:
Gotcha. Grool.

Let's punt on my suggestion for now. We can revisit it later.